docker/api/network.py

```diff
@@ -19,16 +19,30 @@ def networks(self, names=None, ids=None):
         return self._result(res, json=True)
 
     @minimum_version('1.21')
-    def create_network(self, name, driver=None):
+    def create_network(self, name, driver=None, ipam=None):
         data = {
             'name': name,
             'driver': driver,
+            'ipam': ipam,
         }
         url = self._url("/networks/create")
         res = self._post_json(url, data=data)
         return self._result(res, json=True)
 
     @minimum_version('1.21')
+    def generate_ipam_config(self, driver='default', subnet=None,
```

Comment:
Since this doesn't need to exist inside the `Client` (doesn't use `self`), I think it should be moved to a function inside the `docker.utils` module instead.